benchmark_precond_pre_vmult_post/bench.cc

```diff
@@ -0,0 +1,37 @@
+
+#include "../common_code/benchmark.h"
+#include "../common_code/solver_cg_optimized.h"
+
+template <typename Operator, typename Preconditioner>
+std::pair<unsigned int, std::vector<double>>
+run_cg_solver(const Operator &                                  laplace_operator,
+              LinearAlgebra::distributed::Vector<double> &      x,
+              const LinearAlgebra::distributed::Vector<double> &b,
+              const Preconditioner &                            preconditioner)
+{
+  ReductionControl                                              solver_control(100, 1e-15, 1e-8);
+  SolverCGFullMerge<LinearAlgebra::distributed::Vector<double>> solver(solver_control);
```

Comment:
Don't we want 
```suggestion
  SolverCGFullMerge<LinearAlgebra::distributed::Vector<double>> solver(solver_control, false);
```
here?

Reply:
Indeed. Thx!